jenkins/automation/rgw/ceph/ceph-deploy.groovy

```diff
@@ -0,0 +1,153 @@
+pipeline {
+    agent {
+        node {
+            label 'docker-k8-deployment-node'
```

Comment:
can't we use `docker-${os_version}-node` here?

Reply:
Jenkins agent is irrelevant for ceph deployment.

Reply:
Done. using rockylinux-8.4 centos not working.